setup.py

```diff
@@ -176,7 +176,8 @@ def run(self):
                 shutil.copyfile(os.path.join("WebHostLib", "static", "generated", "configs", file_name),
                                 self.buildfolder / "Players" / "Templates" / file_name)
         shutil.copyfile("meta.yaml", self.buildfolder / "Players" / "Templates" / "meta.yaml")
-
+        # TODO: fix LttP options one day
```

Comment:
You know, tracking work in something like github issues is better than random comments in code. ;P